forge-gui/res/cardsfolder/upcoming/gixs_command.txt

```diff
@@ -0,0 +1,15 @@
+Name:Gix's Command
+ManaCost:3 B B
+Types:Sorcery
+A:SP$ Charm | Choices$ DBCounters,DBMurder,DBChangeZone,DBSacrifice | CharmNum$ 2
+SVar:DBCounters:DB$ PutCounter | TargetMin$ 0 | TargetMax$ 1 | ValidTgts$ Creature | TgtPrompt$ Choose target creature | CounterType$ P1P1 | CounterNum$ 2 | SubAbility$ DBPump | SpellDescription$ Put two +1/+1 counters on up to one creature. It gains lifelink until end of turn.
```

Comment:
There is `Choices$` but you might need ChooseCard effect anyway because you need to refer it in the Pump again